dGame/dComponents/ControllablePhysicsComponent.cpp

```diff
@@ -84,14 +86,23 @@ void ControllablePhysicsComponent::Serialize(RakNet::BitStream* outBitStream, bo
 		m_DirtyCheats = false;
 	}
 
-	outBitStream->Write(m_DirtyPickupRadiusScale);
-	if (m_DirtyPickupRadiusScale) {
+	outBitStream->Write(m_DirtyEquippedItemInfo);
+	if (m_DirtyEquippedItemInfo) {
 		outBitStream->Write(m_PickupRadius);
-		outBitStream->Write0(); //No clue what this is so im leaving it false.
-		m_DirtyPickupRadiusScale = false;
+		outBitStream->Write(m_InJetpackMode);
+		m_DirtyEquippedItemInfo = false;
 	}
 
-	outBitStream->Write0();
+	outBitStream->Write(m_BubbleDirty);
+	if (m_BubbleDirty) {
+		outBitStream->Write1(); // another dirty flag?
+		if (m_BubbleDirty) {
+			outBitStream->Write<uint32_t>(1); // 1 not full bubble, 2 full bubble
```

Comment:
should probably have an enum here?

Reply:
Yep, agreed. Haven't actually done work in this since jett, just washed to get the branch updated and merge conflicts resolved so I could start easier lol